Doc/library/subprocess.rst

```diff
@@ -508,12 +512,16 @@ functions.
 
    *stdin*, *stdout* and *stderr* specify the executed program's standard input,
    standard output and standard error file handles, respectively.  Valid values
```

Comment:
Rather than repeating the detailed description in the Popen constructor docs for these handles, end on "respectively. " and just add one additional sentence linking to the frequently-used-arguments section that already documents these semantics.  (deleting the bulk of the original paragraph)